paper-tab.html

```diff
@@ -122,7 +122,8 @@
       },
 
       listeners: {
-        down: '_updateNoink'
+        down: '_updateNoink',
+        click: '_onClick'
```

Comment:
I believe it has to be `click` because that's what iron-button-state uses.

https://github.com/PolymerElements/iron-behaviors/blob/master/iron-button-state.html#L99
https://github.com/PolymerElements/iron-behaviors/blob/master/iron-button-state.html#L176-L180

I'm listening for when someone hits enter or space, which iron-button-state translates into a `click()` call. Would `tap` pick that up as well?